packages/applicant/src/utils/general.ts

```diff
@@ -0,0 +1,5 @@
+const getLoginUrl = () => {
+    return process.env.ONE_LOGIN_ENABLED ? process.env.V2_LOGIN_URL : process.env.LOGIN_URL;
+  };
+
+    export { getLoginUrl };
```

Comment:
Formatting in this file needs fixed up. I've got a PR up to add prettier into our lint config, I'm just fixing that up so I can merge it now.

Reply:
Happy to merge develop into the branch and run prettier when your pr is in

Reply:
PR merged now.